lib/Interpreter/AutoloadCallback.cpp

```diff
@@ -105,14 +105,15 @@ namespace cling {
 
       assert(m_PP);
 
-      auto addFile = [this,decl](llvm::StringRef FileName, bool warn) {
-        if (FileName.empty()) return (const FileEntry*)nullptr;
+      auto addFile = [this,decl](llvm::StringRef FileName, bool warn)
+                                                    -> const clang::FileEntry* {
+        if (FileName.empty()) return nullptr;
 
-        const FileEntry* FE = 0;
+        const clang::FileEntry* FE = 0;
```

Comment:
I chose the same name for a structure in the next PR, and wanted to separate actual behavioral changes with syntax only.

Trying to get to a PR that allows unloading of the printing mechanism and I split it into two parts that are necessary: the original subset of DeclUnloading changes, and the aforementioned one adding **cling::FileEntry**

I guess if the DeclUnloading is delayed, 4e78f9f can just as well be chopped off and I resubmit it with the next PR.